lib/cw.rb

```diff
@@ -0,0 +1,28 @@
+def reverse_inplace(s)
```

Comment:
This is iterative, not recursive... 🤔

Reply:
haha oops, cw.rb is my sandbox file for playing around, not a real hwk submission